blockstream/src/endpoint/height.ts

```diff
@@ -0,0 +1,31 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
```

Comment:
this is nearly identical code to the `difficulty.ts` file...would it make sense to combine them into one to make it more maintainable?

Reply:
I think the general idea is to make them separate to make them maintainable in case the API changes.

Reply:
Yes, good intuition. This is exactly our newest pattern. Will write up a little explanation in a sec.

Reply:
Okay so the new pattern we are trying to run with is:

1. The blockstream External Adapter is a source adapter. It handles READs to go get data from the blockstream API. It is essentially a glorified proxy, with some fancy middleware for caching data. Being a proxy to hit an API we want to keep it as close to the actual API as possible. So we have `endpoints` for the actual API endpoints: `blocks` in this case. By using the `field` parameter we can control which piece of data gets returned from a call to this endpoint.

2. We have these things called an "endpoint" which come in as a parameter from the Chainlink node. These were poorly named and should have been called something like "dataPoint". What it really represents is saying "I want this piece of data for what blockstream says the BTC block difficulty is". The blockstream adapter contains the API specific implementation of how to get that data from blockstream's API.

 In execution this all ends up being just a giant function selector - given some config data go get this data point.